src/main/treeDataProviders/dependenciesTree/dependenciesRoot/nugetTree.ts

```diff
@@ -30,25 +30,20 @@ export class NugetTreeNode extends DependenciesTreeNode {
         }
         for (let key in dependencies) {
             let dependency: any = dependencies[key];
-            let nameVersionTuple: string[] = this.getNameVersionTuple(dependency.dependency.id);
-            let name: string = nameVersionTuple[0];
-            let version: string = nameVersionTuple[1];
-            if (version) {
-                let childDependencies: any = dependency.directDependencies;
-                let generalInfo: GeneralInfo = new GeneralInfo(name, version, '', NugetUtils.PKG_TYPE);
+            let id: string = dependency.id;
+            let version: string = dependency.version;
+            let childDependencies: any = dependency.dependencies;
+            if (id && version && childDependencies) {
```

Comment:
It looks like we filter out nodes without children
```suggestion
            if (id && version) {
```

Reply:
We only filter if one of these fields is null. An empty array is still truthy